exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Packet.java

```diff
@@ -324,7 +329,11 @@ public int getDst_port() {
     byte[] data = null;
     if (packetLength >= payloadDataStart) {
       data = new byte[packetLength - payloadDataStart];
-      System.arraycopy(raw, ipOffset + payloadDataStart, data, 0, data.length);
+      try {
+        System.arraycopy(raw, ipOffset + payloadDataStart, data, 0, data.length);
+      } catch (Exception e) {
```

Comment:
Hi @arina-ielchiieva 
I didn't have a lot of test data to work with, but I do think it would be worth logging the exception.  My general approach was to be as broad as possible so that any parsing error would be caught and the packet would be flagged as corrupt. 

I'll add a logging statement.